dev-workflow-ui/webContent/view/home.xhtml

```diff
@@ -101,7 +101,7 @@
               <p:dataTable id="startedCases" var="case" value="#{homepageIvyDevWfBean.startedCasesDataModel}"
                 widgetVar="casesTable" emptyMessage="No Cases found with given criteria" paginatorPosition="bottom"
                 rows="6" lazy="true" paginator="true" paginatorAlwaysVisible="false" selectionMode="single">
-                <p:ajax event="rowSelect" listener="#{casesIvyDevWfBean.redirectToCaseRow}" />
+                <p:ajax event="rowSelect" listener="#{casesIvyDevWfBean.rerunCaseRow}" />
```

Comment:
I would use case detail as default action. Sorry for switching this many times ;)!